apps/fluent-tester/src/FluentTester/theme/CustomThemes.ts

```diff
@@ -4,10 +4,15 @@ import { ThemeReference } from '@fluentui-react-native/theme';
 import { Platform } from 'react-native';
 import { applyBrand, OfficeBrand } from './applyBrand';
 import { applyTheme, ThemeNames } from './applyTheme';
+import { createAndroidTheme } from '@fluentui-react-native/android-theme';
 
-const themeOptions: ThemeOptions = { paletteName: 'TaskPane', appearance: 'dynamic' };
+const themeOptions: ThemeOptions = Platform.select({
+  android: { appearance: 'dynamic' },
+  default: { paletteName: 'TaskPane', appearance: 'dynamic' },
+});
```

Comment:
Agree about this. Though this brings me to another question - why are the platform specific theming packages separated instead of adding platform specific files and entry point within the default-theme? Is there any specific reason? 👀

Reply:
It came down to the ThemeReference API. The ThemeReference API that the wonderful people like @FalseLobster and @JasonVMo and their teams wrote takes in a "Base Theme", and list of "Theme Recipes" (Either partial themes or functions that return partial themes) to create one final merged theme. The ThemeReference API also lets you specify listeners and ways to invalidate the theme.

When I wrote the Apple Theme, I found that because of the way Native Modules work (this could change for TurboModules), I couldn't just make a ThemeRecipe to layer on the default-theme. I needed a full ThemeReference, complete with it's own baseTheme, partial theme (from the native module) and listeners (either macOS system events or React-Native events on iOS). The ThemeReference API does not let me merge two ThemeReferences together, only a ThemeReference with ThemeRecipes.

Because of that extra level of complexity, I decided to let the AppleTheme be it's own complete ThemeReference that is meant to be used _instead_ of the default-theme, rather than just a ThemeRecipe layered on top. I think this works much better, since the two themes don't have much in common anyway. The only downside is I need to find a way to tell clients "Please use the apple-theme" 😁.

An argument could be made that the default-theme should just return the apple-theme/android-theme on those platforms. I am open to making that change.

Reply:
Oh yeah. I should also mention the win32-theme was implemented as a separate package, so I was copying that approach too.

Reply:
> An argument could be made that the default-theme should just return the apple-theme/android-theme on those platforms. I am open to making that change.

The availability of other platform themes doesn't make sense. Making this change for the android-theme.